sdk/metric/instrument.go

```diff
@@ -77,8 +77,8 @@ type Instrument struct {
 	nonComparable // nolint: unused
 }
 
-// empty returns if all fields of i are their zero-value.
-func (i Instrument) empty() bool {
+// Empty returns if all Instrument fields are their zero-value.
+func (i Instrument) Empty() bool {
```

Comment:
Should this be `IsEmpty()`?  When I read the PR title, I initially interpreted `Empty()` as a verb, and thought it would set all of the fields to their zero-value.

Reply:
SGTM. I'll update.

Reply:
Done.